### ISSUE TITLE
Floating point exception during model evaluation

### ISSUE THREAD

**Issue by cmorganl** [amkozlov/raxml-ng#102]
Hi Alexey,

I've upgraded from RAxML-NG v0.9 to v1.0.0 and I'm suddenly getting a floating point exception error when running `raxml-ng --evaluate`. Here is the whole command (I changed the extensions to 'txt' so I could upload the files):
```bash
raxml-ng --evaluate --msa McrA.fa.txt --tree McrA.nwk.txt --prefix ./tree_data --model LG+G4 --threads 2
```
I've attached the [MSA](https://github.com/amkozlov/raxml-ng/files/5086286/McrA.fa.txt), [tree](https://github.com/amkozlov/raxml-ng/files/5086285/McrA.nwk.txt) and [log](https://github.com/amkozlov/raxml-ng/files/5086283/tree_data.raxml.log) files. I'm running Ubuntu 18.04.

Do you have any idea what may be causing this error to arise? Let me know if you need any more information.

Thanks!
Connor

**Comment by amkozlov** [comment]
Hi Connor,

could you please try adding "--workers 1" option?

Best,
Alexey

**Comment by cmorganl** [comment]
That took care of it, thanks!

Should `--evaluate` always be used with '--workers 1' or is there something else that caused the error?

**Comment by amkozlov** [comment]
no it's probably a bug, I will take care of it once I'm back from vacations!

**Comment by amkozlov** [comment]
turns out this bug has been already fixed (https://github.com/amkozlov/raxml-ng/commit/308ff5cc88d0785fce1308d5953d7b6a644e8cf8), however I made a stupid mistake in packaging: `linux_x86_64` archive used to contain two RAxML-NG binaries: version with the bug (`raxml-ng`) as well as the new/fixed one (`raxml-ng-1.0`). I fixed the archive now, so please update:  

https://github.com/amkozlov/raxml-ng/releases/download/1.0.0/raxml-ng_v1.0.0_linux_x86_64.zip 

Many thanks for reporting!

**Comment by cmorganl** [comment]
Ah, the best kind of bug fix! Thanks, I will update now.
